src/components/ExpensifyButton.js

```diff
@@ -158,6 +167,12 @@ class ExpensifyButton extends Component {
         return (
             <Pressable
                 onPress={this.props.onPress}
+                onLongPress={this.props.onLongPress}
+                onPressIn={() => ControlSelection.block()}
+                onPressOut={() => {
+                    this.props.onPressOut();
+                    ControlSelection.unblock();
+                }}
```

Comment:
Not a good idea. Just move selection blocking logic to the BigNumberPad. We should not extend the scope of this beyond what is needed for this issue.